src/test/run-pass/coherence/re-rebalance-coherence-default-generic-associated-type.rs

```diff
@@ -0,0 +1,22 @@
+#![allow(dead_code)]
+#![feature(re_rebalance_coherence)]
+
+// run-pass
+// aux-build:re_rebalance_coherence_lib.rs
+
+extern crate re_rebalance_coherence_lib as lib;
+use lib::*;
+
+struct MyString {}
+
+impl LibToOwned for MyString {
+    type Owned = String;
+}
+
+impl PartialEq<MyString> for LibCow<MyString> {
```

Comment:
This test doesn't really *check* what the default works out to be, right? Should we do that?